service/src/main.rs

```diff
@@ -690,7 +710,7 @@ fn init_target_parentchain<E>(
 fn init_parentchain<E>(
 	enclave: &Arc<E>,
 	node_api: &ParentchainApi,
-	tee_account_id: &AccountId32,
+	tee_account_id: Option<AccountId32>,
```

Comment:
the idea is: if None, that means that no extrinsics need ever to be sent to this parentchain

Reply:
I think the idea is good, but it should not be documented as a PR comment ;)

Reply:
Same as before, this can also be an `Option<&AccountId32>`.